src/app/problems/problem-view/mcq-view-snippet/mcq-view-snippet.component.ts

```diff
@@ -58,13 +58,18 @@ export class McqViewSnippetComponent implements OnInit {
     /**
      * Submit an answer to the question.
      */
-    onSubmit(formData: { question: number, solution: unknown }): void {
-        this.submissionService.postQuestionSubmission(formData)
+    onSubmit(formData: { question: number, solution: unknown, time_spent: number}): void {
```

Comment:
I don't think `time_spent` is supposed to be in the `formData` type anymore.

Reply:
Didn't catch that one. Thanks for pointing it out. Done.